### PR TITLE
Updated OpLev on Makefile

### PR DESCRIPTION
**Pull request by WilliamChen14**
-O2

### CODE REVIEW

File: Makefile
Comment (andyluo03) [inline Makefile:1]: You needed to pull changes first I think.
Line 1: Use clang++, not g++.
Lines 8: Replace connection.cc with handler.cc. We removed connection files.

File: include/http_request.cc
Comment (andyluo03) [inline include/http_request.cc:1]: Remove this change.

**Review by andyluo03 — CHANGES_REQUESTED** [review]
Address comments then send another request.

**Comment by WilliamChen14** [comment]
I have made the changes requested for the makefile

**Review by andyluo03 — APPROVED** [review]

**Pull request merged**